staff/jordi-pujol/mailchimp-template/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>MailChimp</title>
+</head>
+
+<body>
+    <header class="menu">
+        <nav class="menu__items">
+            <img class="menu__items--title" src="images/main_logo.png">
+            <button class="menu__items--sign" type="submit">Sign Up free</button>
+            <ul class="menu__items--info">
```

Comment:
As a nav bar, if you want a listed one, I think the tags should be: <nav> <ul> <li><a></a></li></ul></nav>

Reply:
Thanks for the feedback, I've just changed it!